test/e2e/network/netpol/network_policy.go

```diff
@@ -350,6 +350,34 @@ var _ = SIGDescribeCopy("Netpol [LinuxOnly]", func() {
 			ValidateOrFail(k8s, model, &TestCase{FromPort: 81, ToPort: 80, Protocol: v1.ProtocolTCP, Reachability: reachability})
 		})
 
+		ginkgo.It("should enforce policy based on multiple label PodSelectors (OR) [Feature:NetworkPolicy]", func() {
+			nsX, nsY, nsZ, model, k8s := getK8SModel(f)
+			peers := []networkingv1.NetworkPolicyPeer{
+				{
+					PodSelector: &metav1.LabelSelector{
+						MatchLabels: map[string]string{"pod": "b"},
+					},
+				}, {
+					PodSelector: &metav1.LabelSelector{
+						MatchLabels: map[string]string{"pod": "c"},
+					},
+				},
+			}
+			policy := GetAllowIngressByMultiLabeledPod("allow-nsx-pod-b-c", map[string]string{"pod": "a"}, peers)
+			CreatePolicy(k8s, policy, nsX)
+
+			reachability := NewReachability(model.AllPods(), true)
+			for _, ns := range []string{nsX, nsY, nsZ} { // Disconnect all pods to Pod a on namespace nsX
```

Comment:
This for-loop can be replaced with `reachability.ExpectAllIngress(NewPodString(nsX, "a"), false)`

Reply:
good suggestions... I'm getting used to it!

Reply:
good suggestions!